test-support/helpers/ember-power-calendar.js

```diff
@@ -1,59 +1,25 @@
-      return target;
-    } else {
-      return find('.ember-power-calendar', target) || find('[data-power-calendar-id]', target);
-    }
-  }
-}
-
-function findCalendarGuid(selector) {
-  let maybeCalendar = findCalendarElement(selector);
-  if (!maybeCalendar) {
-    return;
-  }
-  if (maybeCalendar.classList.contains('ember-power-calendar')) {
-    return maybeCalendar.id;
-  } else {
-    return maybeCalendar.attributes['data-power-calendar-id'].value;
-  }
-}
-
-function findComponentInstance(app, selector) {
-  let calendarGuid = findCalendarGuid(selector);
-  assert(`Could not find a calendar using selector: "${selector}"`, calendarGuid);
-  let calendarService = app.__container__.lookup('service:power-calendar');
-  return calendarService._calendars[calendarGuid];
-}
+import { deprecate } from '@ember/application/deprecations';
+import { calendarSelect, calendarCenter, initCalendarHelpers } from 'ember-power-calendar/test-support';
 
 export default function() {
-  Test.registerAsyncHelper('calendarCenter', async function(app, selector, newCenter) {
-    assert('`calendarCenter` expect a Date or MomentJS object as second argument', newCenter);
-    let calendarComponent = findComponentInstance(app, selector);
-    let onCenterChange = calendarComponent.get('onCenterChange');
-    assert('You cannot call `calendarCenter` on a component that doesn\'t has an `onCenterChange` action', !!onCenterChange);
-    let publicAPI = calendarComponent.get('publicAPI');
-    await publicAPI.actions.changeCenter(newCenter, publicAPI);
-    return wait();
+  Test.registerAsyncHelper('calendarCenter', function(app, selector, newCenter) {
+    deprecate(
+      "The implicit usage of `calendarCenter` will be soon removed. Please switch to `import { calendarCenter } from 'ember-power-calendar/test-support'`",
+      false,
+      { id: 'ember-power-calendar.test-helpers.calendarCenter', until: '1.0.0' }
+    );
+    initCalendarHelpers(app.__container__);
+    return calendarCenter(selector, newCenter);
   });
 
-  Test.registerAsyncHelper('calendarSelect', async function(app, selector, selected) {
-    assert('`calendarSelect` expect a Date or MomentJS object as second argument', selected);
-    let selectedMoment = moment(selected);
-    let calendarElement = findCalendarElement(selector);
-    let daySelector = `${selector} [data-date="${selectedMoment.format('YYYY-MM-DD')}"]`;
-    let dayElement = find(daySelector, calendarElement);
-    if (!dayElement) {
-      await calendarCenter(selector, selected);
-    }
-    return click(daySelector);
+  Test.registerAsyncHelper('calendarSelect', function(app, selector, selected) {
+    deprecate(
+      "The implicit usage of `calendarSelect` will be soon removed. Please switch to `import { calendarSelect } from 'ember-power-calendar/test-support'`",
+      false,
+      { id: 'ember-power-calendar.test-helpers.calendarSelect', until: '1.0.0' }
+    );
+    initCalendarHelpers(app.__container__);
```

Comment:
Do you know if we can use `Ember.getOwner(app)` here?

Reply:
I already tried but seems that app got no owner